Hub doesn't start on Windows due to an error

<!--- Provide a general summary of the issue in the Title above -->

### Steps to reproduce
<!-- Provide an unambiguous set of steps to reproduce the bug -->

1. Build on Windows (`npm run dist:win`)
1. Open the app

### Actual result
<!--- Tell us what happens -->
You'll get an error `Cannot find module 'node-hid'`

### Expected result
<!--- Tell us what should happen -->
The app works.

### Version
<!--- Version and client OS / Branch version -->
1.6.0-beta.1

### Screenshot (if appropriate)
<!--- Please include screenshot capturing UI and open developer tools console -->
<img width="559" alt="screen shot 2018-11-02 at 18 15 39" src="https://user-images.githubusercontent.com/1254342/47985036-bd66f880-e0d8-11e8-98ce-cffe460684c8.png">

--- a/app/src/ledger.js
+++ b/app/src/ledger.js
@@ -1,9 +1,18 @@
 import { app, ipcMain } from 'electron'; // eslint-disable-line import/no-extraneous-dependencies
-import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'; // eslint-disable-line import/no-extraneous-dependencies
 import Lisk from 'lisk-elements'; // eslint-disable-line import/no-extraneous-dependencies
 import { LedgerAccount, SupportedCoin, DposLedger } from 'dpos-ledger-api'; // eslint-disable-line import/no-extraneous-dependencies
 import win from './modules/win';
 
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+// TransportNodeHid = require('@ledgerhq/hw-transport-node-hid');
+// mock transportnodehid, todo fix this to work also on windows and linux
+const TransportNodeHid = {
+  open: () => {},
+  listen: () => {},
+  setListenDevicesDebounce: () => {},
+  setListenDevicesPollingSkip: () => {},
+};
 TransportNodeHid.setListenDevicesDebounce(200);
 /*
 TransportNodeHid.setListenDevicesDebug((msg, ...args) => {

